src/ui/views/Settings.tsx

```diff
@@ -1217,6 +1218,16 @@ options.push(
 		description:
 			"This will automatically delete box scores older than the past three seasons because box scores use a lot of disk space. See Tools > Delete Old Data for more.",
 	},
+	{
```

Comment:
I would leave it off the Settings page and just put an editor directly on the GOAT page, since it only impact that one tangential part of the game, and probably it will always be that way.